content/04.network_applications.md

```diff
@@ -12,6 +12,27 @@
 
 1. Mention techniques for these with some papers
 
+#### Translational Distance Models
+
+Translational distance models view edges in a knowledge graph as linear transformations.
+Every node-edge pair is considered a triplet with head nodes represented as $\textbf{h}$, edges represented as $\textbf{r}$, and tail nodes represented as $\textbf{t}$.
```

Comment:
What about introducing TransE before this as an example. Then this paragraph uses TransE.